src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -1533,7 +1533,7 @@ import TabItem from '@theme/TabItem';
                             
                             rewrite {
                                 activeRecipe("$exampleRecipeName")
-                                exportDatatables = true
+                                exportDatatables(true)
```

Comment:
I believe this should be:

```suggestion
                                setExportDatatables(true)
```

Reply:
That works as well for `build.gradle` at least; probably best to adopt that for `build.gradle.kts` then.

Reply:
Yeah, to make it consistent with both DSLs, we would have to invoke the setter method at this time.